spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin/HttpZipkinSpanReporterTest.java

```diff
@@ -16,10 +18,11 @@
 public class HttpZipkinSpanReporterTest {
 
 	@Rule public final MockWebServer server = new MockWebServer();
+	SpanCollectorMetricsHandler spanCollectorMetricsHandler = new InMemorySpanCollectorMetricsHandler();
```

Comment:
yep. ex in-memory could be in the test tree, as it only really makes sense here.